### ISSUE TITLE
Multi Url Picker property mappings not working in V8. 

### ISSUE THREAD

**Issue by john-mckillip** [AndyButland/UmbracoMapper#15]
I have been playing around with the update for v8. Confirmed that property mappings for Single Url Picker work as expected. When mapping to a Multi Url Picker, the list is always empty in the mapped object.

**Comment by AndyButland** [comment]
Just had a look at this, as I'd not done anything special to handle this data type.  It seems that in V8 we get back a strongly typed model, of type `Umbraco.Web.Models.Link`.  So I find if I have a view model with a property of type `List<Umbraco.Web.Models.Link>`, that does get mapped.

It's mapped to Umbraco's type though, so I'm wondering what if anything to do further here.  For external links, there's not much more value to add - it's already strongly typed.  But maybe for picked pages, to be able to map to your own types there like you can do with collections of `IPublishedContent` - maybe that's worth thinking further about.

**Comment by john-mckillip** [comment]
Ah! So, in my view model, I was mapping to type IEnumerable<Umbraco.Web.Models.Link. I assumed it would be the same as the examples in the ReadMe that show mapping to a collection of custom objects. Switched it to a  List<Umbraco.Web.Models.Link> and it's working as expected. Thanks for looking into it and clarifying!

**Comment by AndyButland** [comment]
Yes, not an unreasonable assumption.  I've got a step in the mapping code where it checks and says, "if I've got back from Umbraco a complex type I don't know about, but it's the same as the view model field I'm mapping to, assign it".  It's just a straight check on the type, so as Umbraco is returning a `List`, it needs to be exactly that.  There might be a more flexible way that would still assign if it was another collection type.
